posthog/api/action.py

```diff
@@ -206,9 +206,9 @@ def _calculate_trends(self, request: request.Request) -> List[Dict[str, Any]]:
     @action(methods=["GET"], detail=False)
     def retention(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
         team = request.user.team_set.get()
-        filter = Filter(request=request)
+        properties = request.GET.get("properties", "{}")
```

Comment:
I'd be great to add a test for this filtering in test_action.py. There are already some tests for retention in there.

Reply:
Sure, I was going to add some frontend tests as well like we have for events.